### DISCUSSION TITLE
Why is Chocolatey enabled by default but not Scoop?

### DISCUSSION THREAD

**Discussion by aaronliu0130** [Devolutions/UniGetUI#669] [General]
At least in my experience, Scoop is a way better experience than Chocolatey. However, Winget selects Chocolatey by default but not Scoop. Besides leading to the weapon selection screen looking weird, there is also no apparent reason to favor Chocolatey. So is there a reason?

**Comment by marticliment** [comment]
The reasons are that:
1) Scoop needs to be installed appart (it cannot be bundled)
2) Scoop software is more specific (cli utilities, linux app ports to windows, etc. It needs to have the extras bucket enabled, which still breaks some compatibility with some apps)
3) Chocolatey has far more packages than scoop does

**Reply by marticliment** [reply]
If scoop was easily-installable without user intervention, it would be also enabled, but since it needs to be installed itself and it needs git to also be installed, I opted to disable it.

**Reply by aaronliu0130** [reply]
What are the difficulties of installing scoop compared to chocolatey? Don't Scoop and Chocolatey both update without git? How would number 2 be a problem? What user intervention is needed to install scoop? I don't know much about this.

**Reply by marticliment** [reply]
The issue is that git also needs to be installed, and this has caused issues with scoop installation script. And not, chocolatey does not use git to update, and it is bundled with wingetui.

There are issues with environment paths, permissions, git itself, and powershell issues with scoop that need to be manually fixed.
